playbook/app/pb_kits/playbook/pb_link/docs/_link_color.html.erb

```diff
@@ -0,0 +1,30 @@
+<div>
+  <%= pb_rails("link", props: {
+    text: "link example",
+    href: "https://www.google.com/",
+  }) %>
+</div>
+
+<div>
+  <%= pb_rails("link", props: {
+    text: "link example",
+    href: "https://www.youtube.com/",
+    color: "body",
+  }) %>
+</div>
+
+<div>
+  <%= pb_rails("link", props: {
+    text: "link example",
+    href: "https://www.facebook.com/",
+    color: "muted",
+  }) %>
+</div>
+
+<div>
+  <%= pb_rails("link", props: {
+    text: "link example",
+    href: "https://www.instagram.com/",
+    color: "destructive",
+  }) %>
```

Comment:
For all your doc examples, let's pick really unique urls, and not share any across multiple sections of docs. That way, it's really clear that a user has never visited that url. 

For many users, when they come to the page, all the links are purple automatically because they've been to youtube.com or google.com etc. 

@kangaree

Reply:
Sounds good! Hah, Ashlee ran into this exact situation.